docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.4"
 
 services:
-  localstack-s3:
+  localstack:
     image: localstack/localstack:latest
     environment:
```

Comment:
there's also `SERVICES=S3` but unsure what it does now:
https://github.com/elifesciences/personalised-covers-formula/blob/6c12a674b8e5f50e2f014340088d42d8007a3ef9/salt/pillar/personalised-covers.sls#L29-L31

Reply:
This limits the services to start up. Updated.